obsidian.css

```diff
@@ -554,61 +554,61 @@ body.plugin-sliding-panes .mod-root .graph-controls {
   border:2px solid transparent;
   transition:none;
 }
-body .nav.svelte-1ocurao.svelte-1ocurao {
```

Comment:
I'm not totally sure what the difference is between this line and the `body .mod-root` line below.